src/main.rs

```diff
@@ -68,12 +71,12 @@ fn app() -> Result {
     let mut cmds = Commands::new();
 
     // Tags
-    cmds.add("?tag {key}", tags::get);
     cmds.add("?tags get {key}", tags::get);
     cmds.add("?tags delete {key}", tags::delete);
     cmds.add("?tags create {key} value...", tags::post);
     cmds.add("?tags get-all", tags::get_all);
```

Comment:
Agreeing with what was said in private, this could be replaced with just `?tags`, while retaining the singular `?tag {key}`.